pyproject.toml

```diff
@@ -0,0 +1,195 @@
+  {name = "Ana Guerrero Lopez" , email = "ana.guerrero@collabora.com"},
+  {name = "Nícolas F. R. A. Prado" , email = "nfraprado@collabora.com"},
+  {name = "kernelci.org bot" , email = "bot@kernelci.org"},
+  {name = "Tyler Baker" , email = "forcedinductionz@gmail.com"},
+  {name = "Laura Nao" , email = "laura.nao@collabora.com"},
+  {name = "Miguel Ojeda" , email = "ojeda@kernel.org"},
+  {name = "khilman" , email = "khilman@users.noreply.github.com"},
+  {name = "Michael Grzeschik" , email = "m.grzeschik@pengutronix.de"},
+  {name = "Alexandra Pereira" , email = "alexandra.pereira@collabora.com"},
+  {name = "Alice Ferrazzi" , email = "alice.ferrazzi@miraclelinux.com"},
+  {name = "Adrian Ratiu" , email = "adrian.ratiu@collabora.com"},
+  {name = "Chris Paterson" , email = "chris.paterson2@renesas.com"},
+  {name = "Guillaume Tucker" , email = "gtucker@gtucker.io"},
+  {name = "Lakshmipathi" , email = "lakshmipathi.ganapathi@collabora.com"},
+  {name = "Lakshmipathi Ganapathi" , email = "lakshmipathi.ganapathi@collabora.com"},
+  {name = "Khouloud Touil" , email = "ktouil@baylibre.com"},
+  {name = "Paweł Wieczorek" , email = "pawiecz@collabora.com"},
+  {name = "Enric Balletbo i Serra" , email = "enric.balletbo@collabora.com"},
+  {name = "Nicolas Belin" , email = "nbelin@baylibre.com"},
+  {name = "Dan Rue" , email = "dan.rue@linaro.org"},
+  {name = "Milo Casagrande" , email = "milo.casagrande@linaro.org"},
+  {name = "Tomeu Vizoso" , email = "tomeu.vizoso@collabora.com"},
+  {name = "Aishwarya TCV" , email = "aishwarya.tcv@arm.com"},
+  {name = "Corentin Labbe" , email = "clabbe@baylibre.com"},
+  {name = "Ricardo Cañuelo" , email = "ricardo.canuelo@collabora.com"},
+  {name = "lakshmipathi.g" , email = "lakshmipathi.ganapathi@collabora.com"},
+  {name = "Denis Efremov" , email = "efremov@linux.com"},
+  {name = "Jeny Sadadia" , email = "jeny.sadadia@gmail.com"},
+  {name = "Lakshmipathi Ganapathi" , email = "lakshmipathi.ganapathi@collabora.co.uk"},
+  {name = "Muhammad Usama Anjum" , email = "usama.anjum@collabora.com"},
+  {name = "Lee Jones" , email = "lee.jones@linaro.org"},
+  {name = "Tzung-Bi Shih" , email = "tzungbi@kernel.org"},
+  {name = "AngeloGioacchino Del Regno" , email = "angelogioacchino.delregno@collabora.com"},
+  {name = "Florent Jacquet" , email = "florent.jacquet@free-electrons.com"},
+  {name = "Jan Luebbe" , email = "jlu@pengutronix.de"},
+  {name = "Michael Walle" , email = "michael@walle.cc"},
+  {name = "Milo Casagrande" , email = "milo@ubuntu.com"},
+  {name = "Arisu Tachibana" , email = "arisu.tachibana@miraclelinux.com"},
+  {name = "Ed Smith" , email = "ed.smith@collabora.com"},
+  {name = "Heiko Thiery" , email = "heiko.thiery@gmail.com"},
+  {name = "Sjoerd Simons" , email = "sjoerd@collabora.com"},
+  {name = "Alex P" , email = "alexandra.pereira@collabora.com"},
+  {name = "Detlev Casanova" , email = "detlev.casanova@collabora.com"},
+  {name = "Guillaume Tucker" , email = "guillaume@mangoz.org"},
```

Comment:
Ha, that's a weird one.  I don't think I've used this address in about 10 years :p  So please drop it as well.